apps/prairielearn/src/tests/groupRolePermissions.test.ts

```
@@ -271,18 +271,20 @@ async function prepareGroup() {
       callback(null);
     });
 
-    after('shut down testing server', helperServer.after);
-    after('unset authenticated user', function (callback) {
+    // shut down testing server
+    afterAll(helperServer.after);
+    // unset authenticated user
+    afterAll(function (callback) {
       Object.assign(config, storedConfig);
       callback(null);
     });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow afterAll duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)